app/controllers/property_or_land/PropertyOrLandAnswerController.scala

```diff
@@ -0,0 +1,95 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.property_or_land
+
+import controllers.actions._
+import controllers.filters.IndexActionFilterProvider
+import javax.inject.Inject
+import models.NormalMode
+import models.Status.Completed
+import navigation.Navigator
+import pages.entitystatus.AssetStatus
+import pages.property_or_land.PropertyOrLandAnswerPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.controller.FrontendBaseController
+import utils.CheckYourAnswersHelper
+import utils.countryOptions.CountryOptions
+import viewmodels.AnswerSection
+import views.html.property_or_land.PropertyOrLandAnswersView
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class PropertyOrLandAnswerController @Inject()(
+                                      override val messagesApi: MessagesApi,
+                                      sessionRepository: SessionRepository,
+                                      navigator: Navigator,
+                                      identify: IdentifierAction,
+                                      getData: DraftIdRetrievalActionProvider,
+                                      requireData: DataRequiredAction,
+                                      requiredAnswer: RequiredAnswerActionProvider,
+                                      validateIndex: IndexActionFilterProvider,
+                                      view: PropertyOrLandAnswersView,
+                                      countryOptions: CountryOptions,
+                                      val controllerComponents: MessagesControllerComponents
+                                     )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+
+  private def actions(index: Int, draftId: String) =
+    identify andThen
+      getData(draftId) andThen
+      requireData // andThen
+//      requiredAnswer(RequiredAnswer(PropertyOrLandAddressPage(index), routes.PropertyOrLandAddressController.onPageLoad(NormalMode, index, draftId)))
+
+
+  def onPageLoad(index: Int, draftId: String): Action[AnyContent] = actions(index, draftId) {
+    implicit request =>
+
+      val answers = new CheckYourAnswersHelper(countryOptions)(request.userAnswers, draftId)
+
+      val sections = Seq(
+        AnswerSection(
+          None,
+          Seq(
+            answers.whatKindOfAsset(index),
+            answers.propertyOrLandAddressYesNo(index),
+            answers.propertyOrLandAddressUkYesNo(index),
+            answers.propertyOrLandUKAddress(index),
+            answers.propertyOrLandInternationalAddress(index),
+            answers.propertyOrLandDescription(index),
```

Comment:
Do we want description in the same line as addressYesNo so it appears in the order they're asked?

Reply:
Description will still appear in the correct order as long as the clean up logic has been applied correctly.

Reply:
Ahh yes! You're correct looking back at the site map again.